### DISCUSSION TITLE
how to map structural descriptor to atomic energy

### DISCUSSION THREAD

**Discussion by J-You** [deepmodeling/deepmd-kit#2494] [Q&A]
Hi all,

I have two questions that are confusing me, and I hope to find solutions:

1. To my understanding, the fitting neural network is a representation of PES from AIMD, and the point in PES should correspond to a frame. The total energy _E_ of a frame is the sum of atomic energy _Ei_ and we use fitting neural network to map structural descriptor _D_ to get atomic energy _Ei_. The fitting neural network should be a supervised model, but how can we acquire the prediction target _Ei_? In training dataset, the energy is the total energy, and actually, _Ei_ cannot be derived from VASP-AIMD simulations.
2. Is it possible to extract the output of embedding net, that is, the structural descriptor _D_ from DPMD-kit?

thanks for any help,
Jiang

**Comment by njzjz** [comment]
>  the fitting neural network is a representation of PES from AIMD,

Actually no. Please read [our paper](https://arxiv.org/pdf/2304.09409.pdf). Eq. 1 and Eq.2 can be a representation of PES. The fitting neural network $\mathcal{F}$ is only a part of Eq. 1 and cannot be considered PES.

> The fitting neural network should be a supervised model, but how can we acquire the prediction target Ei? In training dataset, the energy is the total energy, and actually, Ei cannot be derived from VASP-AIMD simulations.

I guess you mean how to fit $E_i$. Actually, the total energy is used in the loss function instead of $E_i$. See Eq. 60.

> Is it possible to extract the output of embedding net, that is, the structural descriptor D from DPMD-kit?

See #1852.

**Reply by J-You** [reply]
thanks for your reply. 

> I guess you mean how to fit _Ei_ Actually, the total energy is used in the loss function instead of _Ei_. See Eq. 60.

Sure, the predicted total energy is used in loss function, but I am confused how to fit _Ei_, because the true _Ei_ is unknown, at least not in training dataset. The predicted total energy should be the sum of the predicted _Ei_? But I am not sure, it seems also an assumption for extensive character of DP.

**Reply by njzjz** [reply]
The decomposition of $E$ into $E_i$ is arbitrary and there can be countless different combinations (with the same total energies and forces). As a result, $E_i$ must have no physical meaning and thus cannot be fitted alone in theory.

**Reply by J-You** [reply]
Got it! I should look through your published paper. Thanks!
